ggml.h

```diff
@@ -204,6 +204,7 @@ enum ggml_type {
     GGML_TYPE_F16  = 1,
     GGML_TYPE_Q4_0 = 2,
     GGML_TYPE_Q4_1 = 3,
+    GGML_TYPE_Q8_0 = 4,
```

Comment:
I think I want to support it, although I don't expect it to be better than Q4.
It will be a little bit more accurate maybe, but will require much more RAM and disk space.